doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -276,6 +276,7 @@ Machine learning
 ----------------
 
 * Updated the application to enable the :ref:`Zephyr Memory Storage (ZMS) <zephyr:zms_api>` file system for the :ref:`zephyr:nrf54h20dk_nrf54h20` board.
+* Update Edge Impulse URI configuration to use new model location.
```

Comment:
Please remove the changelog entry from this PR, and add it as a suggestion in https://github.com/nrfconnect/sdk-nrf/pull/21339.
```suggestion
* Update the Edge Impulse URI configuration to use the new model location.
```

Reply:
Done